src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="product-page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+  </head>
+  <body class="product-page__body">
+    <div class="product-card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="iMac" 
+        class="product-card__img">
+      <div class="product-card__description">
+        <p class="product-card__description-model">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
```

Comment:
```suggestion
        <h2 class="product-card__description-model">
          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
        </h2>
```
Сделай этот текст заголвоком